core/src/main/java/org/elasticsearch/plugins/PluginsService.java

```diff
@@ -151,7 +164,18 @@ public PluginsService(Settings settings, Path pluginsDirectory, Collection<Class
             }
         }
 
-        logger.info("loaded {}, sites {}", jvmPlugins.keySet(), sitePlugins);
+        // we don't log jars in lib/ we really shouldnt log modules,
+        // but for now: just be transparent so we can debug any potential issues
```

Comment:
Yeah, I think for now i would prefer it at info level only because, this is new and something could go wrong, so it would be at least nice to see modules were loaded on system startup in case someone encounters a problem (the other place we expose modules is in the nodes api for the same reason).

The number of modules is controlled by us and we can change this to debug level at any time we are comfortable.